src/Sentry.AspNetCore/RouteUtils.cs

```diff
@@ -35,7 +35,8 @@ internal static class RouteUtils
             builder.Append(routePattern);
         }
 
-        return builder.ToString();
+        // Force a leading slash (if there isn't already one present)
+        return $"/{builder.ToString().TrimStart('/')}";
```

Comment:
hm this seems lazy, can we check if `/` is there instead of adding and trimming?